docs/ocis/adr/0006-service-discovery.md

```diff
@@ -37,7 +37,7 @@ Chosen option: "Dynamic service registration". There were some drawbacks regardi
 * Having dynamic service registration delegates the entire lifecycle of finding a process to the service registry.
 * Removing a-priori knowledge of hostname + port for services.
 * Marrying go-micro's registry and a newly defined registry abstraction on Reva.
-* We will embrace go-micro interfaces by defining a third merger interface in order to marry go-micro registry and rega revistry.
+* We will embrace go-micro interfaces by defining a third merger interface in order to marry go-micro registry and rega registry.
```

Comment:
Not sure about this. Mostly concerned about `rega`

Reply:
Most likely a reference to github.com/cs3org/reva/ and a fix we can apply ourselves :)

Reply:
fixed s/rega/reva